apiPivotalTest/src/test/resources/cucumber/features/stories/updateStory.feature

```diff
@@ -0,0 +1,32 @@
+@story
+Feature: Update Story
+
```

Comment:
This feature file and deleteStory.feature have the same precondition (Background)
Please join them.